assets/js/src/core.js

```diff
@@ -0,0 +1,40 @@
+function getNewGameUri() {
```

Comment:
It seems really strange to me to have this function rather than a `form`/`submit`. What was the reason for doing it this way?